src/components/tables/Table.tsx

```diff
@@ -117,6 +117,7 @@ export class Table extends React.Component<ITableProps> {
         } as Partial<ITableCompositeState>,
         initialTableData: DEFAULT_TABLE_DATA,
         rowsMultiSelect: false,
+        withoutHoverOnRow: false,

```

Comment:
Shouldn't it be added to `ITableOwnProps` aswell ?

Reply:
nope, this interface extends ITableBodyInheritedFromTableProps with this prop